elastic/elastic-agent
# Fixing Agent README contents

## What does this PR do?



This PR fixes the link to the Agent release notes in the generated README file and removes incorrect instructions copied over from Beats but not applicable to Agent.

## Why is it important?



So users who install Agent and are shown accurate README contents.

## Checklist




## Author's Checklist



## How to test this PR locally



1. Build `elastic-agent`.
   ```
   $ EXTERNAL=true DEV=true SNAPSHOT=true PLATFORMS="<your platform>" mage package
   ```
2. Extract the README from the built artifact.
   ```
   $ tar xvzf build/distributions/elastic-agent-8.8.0-SNAPSHOT-darwin-x86_64.tar.gz elastic-agent-8.8.0-SNAPSHOT-darwin-x86_64/README.md
   ```
4. Check that the links to elastic.co docs in the README have `master` in them, not `main`.
   ```
   grep elastic.co elastic-agent-8.8.0-SNAPSHOT-darwin-x86_64/README.md
   ```

## Related issues


- 

## Use cases



## Screenshots



## Logs
